src/widgets/test-results/overview/helpers.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import styles from "./lab-results.scss";
+import { Information16 } from "@carbon/icons-react";
```

Comment:
For the icons (and the components) - please use submodule imports as shown in one of the recent standups.

--> I think in this codebase we also exclusively should use submodule imports, i.e., imports like `import Information16 from '@carbon/icons-react/es/components/information/16'` or so.